internal/packager/extract.go

```diff
@@ -47,7 +47,7 @@ func findApp() (string, error) {
 }
 
 // extractImage extracts a docker application in a docker image to a temporary directory
-func extractImage(appname string) (string, func(), error) {
+func extractImage(appname string) (string, string, func(), error) {
```

Comment:
We may want to return a struct or something, because 4 return values is starting to be a bit much :sweat: